content/configuration/connectors.md

```diff
@@ -95,6 +96,59 @@ Optional settings:
  through HTTPS, (default: `false`)
  * `timeout` (type _integer_): delay in seconds before declaring a timeout (default: `10`)
 
+## KairosDB
+
+The **KairosDB** connector (type `kairosdb`) can query a [KairosDB REST API][5] to access time series.
+
+*source* and *metric* are modelled distinguishable in KairosDB. The *metric* is accessible directly and the *source* is modelled as a named tag list for each metric. This list defaults to `{"host", "server", "device"}`. Means that *source* is build from metrics `host` tag or `server` tag if `host` is not applied. Because C*/KairosDB could store high frequency series, it is possible to retrieve plots by aggregation. The default aggregation for each metric is `none`. Could be overwritten by `default_aggregator`.
+
+Example *provider* definition using the **KairosDB** connector:
+
+```javascript
+{
+    "connector": {
+        "type": "kairosdb",
+        "url": "http://localhost:8080/",
+        "srctags": [ "host", "name" ],
+        "start_relative": { "value": 6, "unit": "months" },
+        "aggregators": [
+          { "metric": "^entropy\\.",
+            "aggregator": { "name": "min",
+                            "sampling": { "value": 5, "unit": "minutes" } } },
+          { "metric": "\\.[tr]x$",
+            "aggregator": { "name": "max",
+                            "sampling": { "value": 5, "unit": "minutes" } } }
+        ]
+    },
+
+    …
+}
+```
+
+Mandatory settings:
+
+ * `url` (type _string_): URL of the KairosDB REST API (without the `/api/...` path)
+ 
+Optional settings:
+
+ * `srctags` (type _string Array_): Define the metric tags that defines *sources*. Ordering is important. First hit builds the *source*.
```

Comment:
We prefer to use self-descriptive names, could you please s/srctags/source_tags/ ?